config/formats.ts

```diff
@@ -2273,7 +2273,7 @@ export const Formats: (FormatsData | {section: string, column?: number})[] = [
 
 		mod: 'stadium',
 		searchShow: false,
-		ruleset: ['Standard', 'Team Preview'],
+		ruleset: ['Obtainable', 'Team Preview', 'Stadium Sleep Clause Mod', 'Species Clause', 'Nickname Clause', 'OHKO Clause', 'Evasion Moves Clause', 'Endless Battle Clause', 'HP Percentage Mod', 'Cancel Mod'],
```

Comment:
Are all the other rules supposed to be the same? if so, it might be easier to do this:

```suggestion
		ruleset: ['Standard', 'Team Preview', '!Sleep Clause Mod', 'Stadium Sleep Clause Mod'],
```

Reply:
Yes, they should be the same. I didn't know you could do that "!clause" thing, that's convenient!